mobile/test/java/org/chromium/net/impl/CronvoyLoggerTest.java

```diff
@@ -78,10 +74,9 @@ public void testLogToDiskWithLimits() throws Exception {
     String filename = file.getAbsolutePath() + "bar";
     CronvoyLogger logger = new CronvoyLogger();
     logger.setNetLogToDisk(filename, 5);
-    logger.log("hello!");
-    logger.log("goodbye");
+    logger.log(EnvoyLogger.Level.INFO, "hello!");
+    logger.log(EnvoyLogger.Level.INFO, "goodbye");
```

Comment:
Do we need any tets for non-INFO level (I assume that's the default), e.g. log level set to DEBUG and make sure we get `debug` but not `trace` messages?